app/src/main/AndroidManifest.xml

```diff
@@ -12,15 +12,14 @@
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+        android:label="@string/app_name_debug"
```

Comment:
Why we're using `app_name_debug` here? `app_name` is auto generated by gradle and it will point to `app_name_prod` or `app_name_debug` automatically.